cmd/image/import/import.go

```diff
@@ -262,20 +262,23 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --object rhel-83-1003
 			return err
 		}
 		start := time.Now()
-		err = utils.PollUntil(time.Tick(2*time.Minute), time.After(opt.WatchTimeout), func() (bool, error) {
+		var message string
+		err = utils.PollUntil(time.Tick(2*time.Minute), time.After(opt.WatchTimeout), func() (string, bool, error) {
 			job, err := pvmclient.JobClient.Get(*jobRef.ID)
 			if err != nil {
-				return false, fmt.Errorf("image import job failed to complete, err: %v", err)
+				message = fmt.Sprintf("image import job failed to complete, err: %v", err)
+				return message, false, fmt.Errorf("image import job failed to complete, err: %v", err)
 			}
 			if *job.Status.State == jobStateCompleted {
-				klog.V(2).Infof("Image uploaded successfully, took %s", time.Since(start).Round(time.Second))
-				return true, nil
+				message = fmt.Sprintf("Image uploaded successfully, took %s", time.Since(start).Round(time.Second))
+				return message, true, nil
 			}
 			if *job.Status.State == jobStateFailed {
-				return false, fmt.Errorf("image import job failed to complete, err: %v", job.Status.Message)
+				message = fmt.Sprintf("image import job failed to complete, err: %v", job.Status.Message)
+				return message, false, fmt.Errorf("image import job failed to complete, err: %v", job.Status.Message)
```

Comment:
Same as below, let's not send a message to display from the ticker and log using `klog.Errorf` to display the error.

Reply:
Added above changes